packages/lesswrong/lib/routes.ts

```diff
@@ -533,74 +547,22 @@ if (taggingNameIsSet.get()) {
     //   background: "white",
     //  getPingback: (parsedUrl) => getTagPingbackBySlug(parsedUrl, parsedUrl.params.slug),
     // },
+    // {
+    //   name: 'tag.arbital.p',
+    //   path: '/p/:slug',
```

Comment:
I'm pretty sure there are some `/p/` routes in the code that need to be refactored

Reply:
(Also I'm pretty sure there are more `/tag/` routes that this didn't catch)

Reply:
Also there are several uses of `taggingNamePluralSetting` in various components/other files for link construction, not just text display, which probably make sense to change as well